docs/input/docs/reference/configuration.md

```diff
@@ -53,6 +53,7 @@ commit-date-format: yyyy-MM-dd
 merge-message-formats: {}
 update-build-number: true
 semantic-version-format: Strict
+version-strategy: NonTrunkBased
```

Comment:
Can we just call the property `strategy`?

```suggestion
strategy: NonTrunkBased
```

Reply:
I name it strategies intead because it is an array data type.

Reply:
done